public/src/pages/views/LabResultView.tsx

```diff
@@ -54,8 +55,8 @@ export class LabResultView extends React.Component<ILabInfoProps> {
     }
 
     private getSubmissionInfo(): string {
-        if (this.props.labInfo.latest) {
-            return this.props.labInfo.latest.approved ? "Approved" : "Not approved";
+        if (this.props.assignment.latest) {
+            return this.props.assignment.latest.approved ? "Approved" : "Not approved";
```

Comment:
Not Approved (capital A)

Reply:
The method was not even used anymore, so I've just removed it.